### PR TITLE
makes the css for the promo banner critical to prevent a loading-time…

### PR DESCRIPTION
**Pull request by pixelandpage**
Makes the css for the promo banner 'critical' to prevent a loading-time bug on slow devices. 

slack_for_ios_upload

### CODE REVIEW

File: myft-digest-promo/main.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@include nUiMaybeCriticalStart('myftDigestPromo');
```
Comment (debugwand) [inline myft-digest-promo/main.scss]: there should be a matching nUiMaybeCriticalEnd

**Comment by wheresrhys** [comment]
There might be a better way to achieve this, will come to talk it over tomorrow

File: myft-digest-promo/main.scss (hunk 1 above)
Comment (pixelandpage) [inline myft-digest-promo/main.scss]: Good idea!

**Comment by keirog** [comment]
I‘m confused by the absence of [`nUiApply`](https://github.com/Financial-Times/n-ui/blob/master/tour-tip/main.scss#L3) here, but I don‘t really understand how n-ui+Sass works 😕

**Comment by pixelandpage** [comment]
Latest commit makes better use of nUiCritical and nUiApply. It also moves the promo init() JS function call to delay it until after the non-critical CSS has loaded.

File: myft-digest-promo/main.scss [hunk 2]
```diff
@@ -1,108 +1,115 @@
...
+		float: none;
 		position: relative;
-		@include oGridRespondTo('M') {
-			@include oGridCenter;
-			@include oGridOffset(2);
+		background-color: getColor('warm-5');
+		border-bottom: 1px solid getColor('warm-3');
+		border-top: 1px solid getColor('warm-3');
+
+		&.n-myft-digest-promo--enabled {
```
Comment (keirog) [inline myft-digest-promo/main.scss]: We shouldn’t need to chain the classes here; should be ok to have the `.n-myft-digest-promo--enabled` as its own block, outside and under this `.n-myft-digest-promo` block

**Pull request merged**